cdp_scrapers/legistar_utils.py

```diff
@@ -677,11 +946,12 @@ def get_person(self, legistar_person: Dict) -> Optional[Person]:
         return self.get_none_if_empty(
             Person(
                 email=str_simplified(legistar_person[LEGISTAR_PERSON_EMAIL]),
-                external_source_id=legistar_person[LEGISTAR_PERSON_EXT_ID],
+                external_source_id=str(legistar_person[LEGISTAR_PERSON_EXT_ID]),
                 name=str_simplified(legistar_person[LEGISTAR_PERSON_NAME]),
                 phone=phone,
                 website=str_simplified(legistar_person[LEGISTAR_PERSON_WEBSITE]),
                 is_active=bool(legistar_person[LEGISTAR_PERSON_ACTIVE]),
+                roles=self.get_roles(legistar_person[LEGISTAR_PERSON_ROLES]),
```

Comment:
This should be removed. I will fix upstream but `roles` should be under `seat`

In the future this should be `seat=Seat(..., roles=self.get_roles(...))`

and by "future" i mean in like 20 minutes when I release v3.0.0.dev15